packages/react/src/components/cookieConsent/contexts/ContentContext.tsx

```diff
@@ -73,11 +73,13 @@ export type Callbacks = {
   onConsentsParsed?: (consents: ConsentObject, hasUserHandledAllConsents: boolean) => void;
 };
 
+/* eslint-disable */
 type ConsentContextProps = {
   cookieDomain?: string;
   children: React.ReactNode | React.ReactNode[] | null;
   contentSource: CookieContentSource;
 };
+/* eslint-enable */
```

Comment:
Does this part mess up eslint for some reason? 😆

Reply:
It should not. It just enables it again after a few lines of being disabled

Reply:
Yes I meant the disabled part that it's funny and weird that apparently it enrages eslint. Nothing wrong with this, I was just curious :)